bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/help/WorkbenchHelpSystem.java

```diff
@@ -552,7 +552,7 @@ private HelpListener getHelpListener() {
 	 *             {@link org.eclipse.help.HelpSystem HelpSystem}instead of the
 	 *             IHelp methods on the object returned by this method.
 	 */
-	@Deprecated
+	@Deprecated(forRemoval = true)
```

Comment:
Please add since= "2023-12" . This would make it obiouls that it can be removed for 2025-12 release.

Reply:
done, here ant there: https://github.com/eclipse-platform/eclipse.platform/pull/773